src/mscorlib/src/System/Runtime/Loader/AssemblyLoadContext.cs

```diff
@@ -396,6 +398,9 @@ public void StartProfileOptimization(string profile)
         // specified by the host. By having the field as a static, we are
         // making it an AppDomain-wide field.
         private static volatile AssemblyLoadContext m_DefaultAssemblyLoadContext;
+
+        // Synchronization primitive for controlling initialization of Default load context
+        private static object m_initLock = new Object();
```

Comment:
(It's also a bit strange to me that ```m_``` is used as a prefix for statics, but I see that's already done elsewhere in the file.)